app/api/v1/access.py

```
@@ -97,7 +97,7 @@ async def add_vault_member(
     except ValueError as e:
         # e.g., attempting to add the owner as a member
         print(f"Error: {e}")
-        raise HTTPException(status_code=status.HTTP_422_UNPROCESSABLE_ENTITY, detail=str(e))
+        raise HTTPException(status_code=status.HTTP_422_UNPROCESSABLE_CONTENT, detail=str(e))
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)